src/main/kotlin/com/github/blarc/ai/commits/intellij/plugin/settings/AppSettings2.kt

```diff
@@ -59,12 +60,14 @@ class AppSettings2 : PersistentStateComponent<AppSettings2> {
             GeminiClientConfiguration::class,
             AnthropicClientConfiguration::class,
             AzureOpenAiClientConfiguration::class,
-            HuggingFaceClientConfiguration::class
+            HuggingFaceClientConfiguration::class,
+            OpenRouterClientConfiguration::class
         ],
         style = XCollection.Style.v2
     )
     var llmClientConfigurations = setOf<LLMClientConfiguration>(
-        OpenAiClientConfiguration()
+        OpenAiClientConfiguration(),
+        OpenRouterClientConfiguration()
```

Comment:
This is not necessary. Users can create new configurations via the add button. OpenAiClientConfiguration is added just as an example.